lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+    type: int
+  cpu_shares:
+    description:
+      - CPU shares (relative weight)
+    type: int
+  cpus:
+    description:
+      - Number of CPUs. The default is 0.0 which means no limit.
+    type: str
+  cpuset_cpusr:
+    description:
+      - CPUs in which to allow execution (0-3, 0,1)
+    type: str
+  cpuset_mems:
+    description:
+      - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
+    type: str
+  detach:
+    description:
+      - Run container in detach mode
+    type: bool
+    default: True
+  detach_keys:
+    description:
+      - Override the key sequence for detaching a container. Format is a single character or ctrl-value
+    type: str
+  device:
+    description:
+      - Add a host device to the container.
+        The format is <device-on-host>[:<device-on-container>][:<permissions>]
+        (e.g. device /dev/sdc:/dev/xvdc:rwm)
+    type: str
+  device_read_bps:
+    description:
+      - Limit read rate (bytes per second) from a device (e.g. device-read-bps /dev/sda:1mb)
+    type: str
+  device_read_iops:
+    description:
+      - Limit read rate (IO per second) from a device (e.g. device-read-iops /dev/sda:1000)
+    type: str
+  device_write_bps:
+    description:
+      - Limit write rate (bytes per second) to a device (e.g. device-write-bps /dev/sda:1mb)
+    type: str
+  device_write_iops:
+    description:
+      - Limit write rate (IO per second) to a device (e.g. device-write-iops /dev/sda:1000)
+    type: str
+  dns:
+    description:
+      - Set custom DNS servers
+    type: list
+  dns_option:
+    description:
+      - Set custom DNS options
+    type: str
+  dns_search:
+    description:
+      - Set custom DNS search domains (Use dns_search with '' if you don't wish to set the search domain)
+    type: str
+  entrypoint:
+    description:
+      - Overwrite the default ENTRYPOINT of the image
+    type: str
+  env:
+    description:
+      - Set environment variables.
+        This option allows you to specify arbitrary environment variables that
+        are available for the process that will be launched inside of the container.
+    type: dict
+  env_file:
+    description:
+      - Read in a line delimited file of environment variables
+    type: str
+  etc_hosts:
+    description:
+      - Dict of host-to-IP mappings, where each host name is a key in the dictionary.
+          Each host name will be added to the container's ``/etc/hosts`` file.
+    type: dict
+    aliases:
+      - add_hosts
+  expose:
+    description:
+      - Expose a port, or a range of ports (e.g. expose "3300-3310") to set up
+        port redirection on the host system.
+    type: list
+    aliases:
+      - exposed
+      - exposed_ports
+  force_restart:
+    description:
+      - Force restart of container.
+    type: bool
+    aliases:
+      - restart
+  gidmap:
+    description:
+      - Run the container in a new user namespace using the supplied mapping.
```

Comment:
You should provide an example. Also, I would expect this to be of type `dict`, not `str`.